### DISCUSSION TITLE
Allow customization of transition of primereact's dialog

### DISCUSSION THREAD

**Discussion by Et7f3** [primefaces/community#1977] [PrimeReact]
I love the transition and wanted to configure it but it seems hard coded. Does dialog have a way to customize the transition ? https://github.com/primefaces/primereact/blob/22c459a4c580a3abc1dee70399fa57415ccd2e39/components/lib/dialog/DialogBase.js#L114

**Comment by ussfranck** [comment]
### **PrimeReact v10.9.1**
Hello. I agree with you, it's complicated to overload a personalized transition.
The way I've found that's so unadvisable is to identify all the css classes linked to a component, then override them.
